ecs-proxies-deploy.yml

```diff
@@ -3,4 +3,4 @@ docker_service:
     expose: true
```

Comment:
This was causing the sandbox to constantly be restarted and is probably the reason we have been having intermittent problems with responses from the sandbox for some time